forge-game/src/main/java/forge/game/player/PlayerProperty.java

```diff
@@ -373,6 +373,10 @@ else if (kind.startsWith("Type")) {
             if (!AbilityUtils.getDefinedPlayers(source, property, spellAbility).contains(player)) {
                 return false;
             }
+        } else if (property.startsWith("NonTriggered")) {
```

Comment:
This shouldn't be needed, you're not using any properties

Reply:
Ah right because it's Defined

Thank you